src/components/History.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const History = ({ history }) => {
+  const mappedHistory = history.map(past => {
+    return (
+      <li key={Math.random()}>
```

Comment:
it's better to set the key as the index of the past in the array. Math.random will not reliably give you a unique key. Index isn't a great option, but you should use it when you don't have a better option.